### PR TITLE
Tlunizin-596 Modify sis upload script so that practice courses are created in the Practice Course sub-account

### PR DESCRIPTION
**Pull request by zqian**
separate the sandbox sites creation routine into different file;
improve logger logging messages.

### CODE REVIEW

File: SIS_import/sis_upload.rb [hunk 1]
```diff
@@ -106,15 +101,15 @@ def Canvas_API_POST(url, params, fileName)
...
-																	:verify_ssl => true}
+			response = RestClient.post Addressable::URI.escape(url),
+			                           {:multipart => true,
+			                            :attachment => File.new(fileName, 'rb')
+			                           },
+			                           {:Authorization => "Bearer #{$token}",
+			                            :accept => "application/json",
+			                            :import_type => "instructure_csv",
+			                            :content_type => "application/zip",
+			                            :verify_ssl => true}
```
Comment (dovek) [inline SIS_import/sis_upload.rb]: Are these indentations intended? Or is this a result of the IDE you were using to write the code?

File: SIS_import/sis_instructor_practice_course.rb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+			sandbox_course_id = result.fetch("id")
+			instructor_result = Canvas_API_POST("#{server_api_url}courses/#{sandbox_course_id}/enrollments",
+			                                    {
+				                                    "enrollment[user_id]" => user_canvas_id,
+				                                    "enrollment[type]" => "TeacherEnrollment",
+				                                    "enrollment[enrollment_state]" => "active"
+			                                    },
+			                                    nil)
+			logger.info "Enrolled User #{user_sis_login_id} to sandbox course site - #{user_sandbox_site_name} (#{sandbox_course_id}) \n #{instructor_result} \n"
+		end
```
Comment (pushyamig) [inline SIS_import/sis_instructor_practice_course.rb]: What if result.has_key?("id") don't have Id, it should be logged at least

File: SIS_import/sis_upload.rb (hunk 1 above)
Comment (zqian) [inline SIS_import/sis_upload.rb]: I added those indentation to increase readability. I see the problem on line 106. Fixed.

File: SIS_import/sis_instructor_practice_course.rb (hunk 2 above)
Comment (zqian) [inline SIS_import/sis_instructor_practice_course.rb]: Log as warning.

**Pull request merged**